code/modules/events/wizard/madness.dm

```diff
@@ -12,7 +12,7 @@
 
 	var/suggested = pick(strings(REDPILL_FILE, "redpill_questions"))
 
-	forced_secret = (input(usr, "What horrifying truth will you reveal?", "Curse of Madness", suggested) as text) || suggested
+	forced_secret = (input(usr, "What horrifying truth will you reveal?", "Curse of Madness", suggested) as text|null) || suggested
```

Comment:
will be `suggested` if cancelled instead of aborting. possibly fine though?

Reply:
think this is fine